rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/rules/CreateAboutWindupReportRuleProvider.java

```diff
@@ -85,7 +85,7 @@ private ApplicationReportModel createApplicationReport(GraphContext context, Pro
         applicationReportModel.setDisplayInApplicationReportIndex(true);
         applicationReportModel.setReportName(REPORT_NAME);
         applicationReportModel.setReportIconClass("glyphicon glyphicon-info-sign");
-        applicationReportModel.setMainApplicationReport(true);
+        applicationReportModel.setMainApplicationReport(false);
```

Comment:
this is the main fix because of how method "ApplicationReportService.getMainApplicationReportForFile" works